content/2017/1/26/we-are-not-neutral.rst

```diff
@@ -0,0 +1,110 @@
+  We can't distance ourselves from the world we work in,
+  or the lives of our members, colleagues, and clients.
+  Software is never neutral,
+  and neither are we.
+  There's work to do.
+
+
+We Are Not Neutral: 2016 in Review
+==================================
+
+OddBird is a small company —
+a shared vision and brand
+that transforms seven remote contractors
+into a *Web Software Agency*.
+
+2016 was a year of big changes for us.
+`Carl`_ took a job with `Instagram`_,
+`Sondra`_ came on full-time,
+and we added `David`_,
+`Stacy`_,
+and `Kit`_ to the team.
+We're `writing more`_,
+`speaking more`_,
+`taking on more clients`_,
+`re-designing our website and brand in the open`_,
+`building a community on Slack`_,
+and contributing to more `open-source projects`_.
+
+.. _`Carl`: /birds/#bird-carl
+.. _`Instagram`: http://instagram.com
+.. _`Sondra`: /birds/#bird-sondra
+.. _`David`: /birds/#bird-david
+.. _`Stacy`: /birds/#bird-stacy
+.. _`Kit`: /birds/#bird-kit
+.. _`writing more`: /blog/
+.. _`speaking more`: /speaking/
+.. _`taking on more clients`: /work/
+.. _`re-designing our website and brand in the open`: /tags/open-design/
+.. _`building a community on Slack`: http://friends.oddbird.net/
+.. _`open-source projects`: /open-source/
+
+It was hard to lose one of the founders
+(and a sibling!)
+from our daily work,
+but we're glad that he still drops by —
+an active outside eye when we need advice.
+This is the first time our founders have been outnumbered,
+but I couldn't ask for a better team.
+They all have impressive skills,
+but more importantly:
+they are all kind,
+empathetic, thoughtful, and generous.
+To me,
+these are not only the most important qualities
+in a colleague I see every day,
+but the most important skills
+for any software developer.
+
+We know that our tools have an effect
+on the lives of the people we work with,
+in real and lasting ways.
+We know that software is never neutral,
+and assumptions we make as developers
+can `cause real pain`_ if
+we aren't thoughtful about every aspect
+of design and development.
+We have a responsibility to consider
+how our products are used by diverse people,
+long after we move on.
+We can't build for the center;
+we have to build for the edges.
+
+.. _`cause real pain`: http://meyerweb.com/eric/thoughts/2014/12/24/inadvertent-algorithmic-cruelty/
+
+OddBird may not be a political organization,
+but we can't distance ourselves from the world we work in,
+or the lives of our members, colleagues, and clients.
+2016 was a hard year for many us,
+and the people we care about.
+We expect 2017 to continue that trend —
+with threats to health insurance,
+immigrants, people of color, lgbtqai+ people,
+muslims, women, native people, and so on.
+
+I don't have any magic words,
+but it's important to all of us here
+that OddBird double-down on our commitment
+to *people* in all their diversity,
+especially when they are pushed to the margins.
+We know that we have work to do,
```

Comment:
A comma doesn't seem quite right here, but I'm not sure if `;` or `--` would be better?